### DISCUSSION TITLE
3.7.1

### DISCUSSION THREAD

**Discussion by patrick96** [polybar/polybar#3053] [Announcements]
### Build
- Fixed missing header when using `libc++` in clang 15 and below

### Changed
- `internal/tray`: The module must use the `<tray>` tag (this is the default) ([`#3037`](https://github.com/polybar/polybar/pull/3037))

### Fixed
- Modules did not validate that all tags (e.g. `<label>`) used in a format were valid for that format ([`#3043`](https://github.com/polybar/polybar/issues/3043), [`#3045`](https://github.com/polybar/polybar/pull/3045))
- `internal/tray`: Fixed `module-margin` and `separator` being applied to completely empty tray module ([`#3036`](https://github.com/polybar/polybar/issues/3036), [`#3037`](https://github.com/polybar/polybar/pull/3037))

**Full Changelog**: https://github.com/polybar/polybar/compare/3.7.0...3.7.1

<hr /><em>This discussion was created from the release <a href='https://github.com/polybar/polybar/releases/tag/3.7.1'>3.7.1</a>.</em>

**Comment by mbrunnen** [comment]
Isn’t this a breaking change, i.e. 4.0.0?

**Reply by patrick96** [reply]
Is what a breaking change?
